File: articles/what-is-analytical-psychology/go/compensation_pressure_score.go
package main

import "fmt"

func compensationPressure(
	consciousOneSidedness,
	unconsciousActivation,
	symbolicIntensity,
	regulatoryCapacity float64,
) float64 {
	return consciousOneSidedness +
		unconsciousActivation +
		symbolicIntensity -
		regulatoryCapacity
}

func main() {
	score := compensationPressure(0.82, 0.74, 0.68, 0.36)
	fmt.Printf("Synthetic compensation pressure: %.3f\n", score)
}

File: articles/what-is-analytical-psychology/fortran/compensation_pressure_score.f90
! What Is Analytical Psychology?
! Fortran example: simple synthetic compensation-pressure score

program compensation_pressure_score_demo
  implicit none

  real :: score

  score = 0.82 + 0.74 + 0.68 - 0.36

  print *, "Synthetic compensation pressure:", score
end program compensation_pressure_score_demo
